lib/helpers.js

```diff
@@ -11,8 +12,7 @@ export function notifyAllowWallet({account, preventRenotify = false}) {
   if(preventRenotify && notifiedAccounts.get(account)) {
     return;
   }
-  // if inside a WebView ignore
-  if(isWebView({userAgent: navigator?.userAgent})) {
+  if(config?.vueWallet?.disableChapi === true) {
```

Comment:
I think we can assume both `config` and `config.vueWallet` are defined -- we probably make that assumption in a number of places in this module already. +1 to simplifying by removing `=== true` as well.

```suggestion
  if(config.vueWallet.disableChapi) {
```